example-plugin/build.gradle

```diff
@@ -32,7 +32,7 @@ dependencies {
     compileOnly "com.yelp.nrtsearch:server:${nrtsearchVersion}"
     testImplementation "com.yelp.nrtsearch:server:${nrtsearchVersion}"
     testImplementation("com.yelp.nrtsearch:server:${nrtsearchVersion}:tests")
-    testImplementation "junit:junit:4.13"
+    testImplementation "junit:junit:4.13.2"
```

Comment:
Can you just apply the version here from the other file, where you set the junit version?

Reply:
The problem is that `example-plugin` is structured as its own gradle project, so it does not have access to variables from the other project.

Reply:
LGTM.